### PR TITLE
Chunter/Task project config.

### PR DESCRIPTION
**Pull request by haiodo**
1. Allow to edit Project and Channel details.
2. Change Dialog style
3. Fix ordering issue with memdb.
4. Some minor improvements.

Screenshot 2021-11-04 at 20 58 53

Screenshot 2021-11-04 at 20 58 58

### CODE REVIEW

File: plugins/task-impl/src/components/StatusPicker.svelte [hunk 1]
```diff
@@ -13,54 +13,34 @@
...
-        }}
-      />
-    {/each}
-  </PopupMenu>
   <div
+    bind:this={btn}
     class="selected"
     on:click|preventDefault={(event) => {
-      pressed = !pressed
+      console.log('pressed, pressed')
```
Comment (BykhovDenis) [inline plugins/task-impl/src/components/StatusPicker.svelte:32]: Extra log

File: plugins/workbench-impl/src/components/Applications.svelte [hunk 2]
```diff
@@ -34,6 +34,7 @@
 
   query = client.query(query, workbench.class.Application, {}, (result) => {
     apps = result
+    console.log('applications', apps)
```
Comment (BykhovDenis) [inline plugins/workbench-impl/src/components/Applications.svelte:37]: Extra log

File: plugins/workbench-impl/src/components/navigator/SpacesNav.svelte [hunk 3]
```diff
@@ -121,6 +120,30 @@
     return result
   }
 
+  function spaceActions (model: SpacesNavModel, space: Space): Action[] {
+    const result: Action[] = []
+
+    console.log(model.spaceMore)
```
Comment (BykhovDenis) [inline plugins/workbench-impl/src/components/navigator/SpacesNav.svelte:126]: Extra log

File: plugins/workbench-impl/src/components/MembersSection.svelte [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    members = m
+  })
+
+  $: client.findAll(core.class.Account, {}).then((m) => {
+    otherMembers = m.filter((m) => !space.members.includes(m._id))
+  })
+  let btn: HTMLElement | undefined
+
+  function inviteMember (): void {
+    console.log(members, otherMembers)
```
Comment (BykhovDenis) [inline plugins/workbench-impl/src/components/MembersSection.svelte:24]: Extra log

**Review by BykhovDenis — APPROVED** [review]
Please remove extra logs. Looks nice

File: plugins/workbench-impl/src/components/Applications.svelte (hunk 2 above)
Comment (haiodo) [reply plugins/workbench-impl/src/components/Applications.svelte]: Fixed

File: plugins/workbench-impl/src/components/navigator/SpacesNav.svelte (hunk 3 above)
Comment (haiodo) [reply plugins/workbench-impl/src/components/navigator/SpacesNav.svelte]: Fixed

File: plugins/workbench-impl/src/components/MembersSection.svelte (hunk 4 above)
Comment (haiodo) [reply plugins/workbench-impl/src/components/MembersSection.svelte]: Fixed

File: plugins/task-assets/assets/icons.svg [hunk 5]
```diff
@@ -13,4 +13,9 @@
     />
   </symbol>
 
+  <symbol id="project" viewBox="0 0 20 20" fill='none'>
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M16.9796 12.6276C16.9796 15.0397 15.4415 16.4615 12.8322 16.4615H7.1569C4.54097 16.4615 3 15.0397 3 12.6276V7.36908C3 4.96028 3.95827 3.53847 6.56837 3.53847H8.02693C8.55055 3.53914 9.04355 3.76634 9.35714 4.15398L10.023 4.97242C10.338 5.35939 10.831 5.58726 11.3546 5.58793H13.4185C16.0344 5.58793 17 6.81896 17 9.28033L16.9796 12.6276Z" stroke="#ffffff" stroke-linecap="round" stroke-linejoin="round"/>
+    <path d="M6.51172 11.916H13.2738" stroke="#ffffff" stroke-linecap="round" stroke-linejoin="round"/>
```
Comment (SasLord) [inline plugins/task-assets/assets/icons.svg:18]: stroke - вот такого в иконках не должно быть. :) Я переделаю.

**Review by SasLord — APPROVED** [review]

**Pull request merged**